src/General/Store.hs

```diff
@@ -63,9 +68,9 @@ decodeBS = decode . LBS.fromStrict
 
 -- each atom name is either unique (a scope) or "" (a list entry)
 data Atom = Atom
-    {atomType :: String -- Type that the atom contains (for sanity checking)
+    {atomType     :: String -- Type that the atom contains (for sanity checking)
```

Comment:
I don't care about the imports of extensions side of thing, but indenting stuff to align up with `::` is pretty annoying as it makes diffs larger. If you added a stylish.yaml file or similar in the root of Hoogle could we disable that aspect? If so, happy to add that config file.